Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/StatisticsExplanationFragment.kt

```diff
@@ -28,6 +28,12 @@ class StatisticsExplanationFragment : Fragment(R.layout.fragment_statistics_expl
             R.string.statistics_explanation_seven_day_r_link_label,
             R.string.statistics_explanation_faq_url
         )
+
+        binding.statisticsExplanationTrendText.apply {
+            val label = String.format(getString(R.string.statistics_explanation_trend_text))
+            text = label
+            contentDescription = label
```

Comment:
You just adapted what was there in XML, but why do we even need `contentDescription` on a `TextView` without any difference to the text?.

Reply:
Is it due to?
> Check the description and make sure it says "...1%..." and "...5%..."

Reply:
In this case, yes.